lib/db.js

```diff
@@ -365,8 +361,9 @@ DB.prototype._revisionPolicyUpdate = function(tableName, query, schema) {
                     attributes: item
                 };
                 updateQuery.attributes._exist_until = expireTime;
-                return dbu.buildPutQuery(updateQuery, tableName, schema).data;
+                return dbu.buildPutQuery(updateQuery, tableName, schema, true);
             }))
+            .reduce(function(accumulator, item) { return accumulator.concat(item); }, [])
```

Comment:
`P.all` here returns an array of arrays. `reduce` flatters that structure into a one-level array.